core/src/main/scala/org/apache/spark/storage/BlockManagerMaster.scala

```diff
@@ -167,10 +167,12 @@ class BlockManagerMaster(
    * amount of remaining memory.
    */
   def getMemoryStatus: Map[BlockManagerId, (Long, Long)] = {
+    if (driverEndpoint == null) return Map.empty
```

Comment:
Although only `getStorageStatus` is used, I added the preventive code into `getMemoryStatus` together.